powerlevel9k.zsh-theme

```diff
@@ -497,13 +496,43 @@ prompt_dir() {
           current_path=$(truncatePathFromRight "$(pwd | sed -e "s,^$HOME,~,")" )
         fi
       ;;
+      truncate_with_folder_marker)
+        local last_marked_folder marked_folder zero
+        set_default POWERLEVEL9K_SHORTEN_FOLDER_MARKER ".shorten_folder_marker"
+
+        for marked_folder in $(upsearch $POWERLEVEL9K_SHORTEN_FOLDER_MARKER); do
+          if [[ "$marked_folder" == "/" ]]; then
+            current_path="/"
+          elif [[ "$marked_folder" == "$HOME" ]]; then
+            current_path="~"
+          elif [[ "${marked_folder%/*}" == $last_marked_folder ]]; then
+            current_path="${current_path%/}/${marked_folder##*/}"
+          else
+            current_path="${current_path%/}/$POWERLEVEL9K_SHORTEN_DELIMITER/${marked_folder##*/}"
+          fi
+          last_marked_folder=$marked_folder
+        done
+
+        zero='%([BSUbfksu]|([FB]|){*})'
+        current_path=$current_path${PWD:${#${(S%%)last_marked_folder//$~zero/}}}
+      ;;
       *)
         current_path="%$((POWERLEVEL9K_SHORTEN_DIR_LENGTH+1))(c:$POWERLEVEL9K_SHORTEN_DELIMITER/:)%${POWERLEVEL9K_SHORTEN_DIR_LENGTH}c"
       ;;
     esac
 
   fi
 
+  set_default POWERLEVEL9K_DIR_SPLIT_MODE false
+  if [[ $POWERLEVEL9K_DIR_SPLIT_MODE == true ]]; then
+    current_path=$(print -P $current_path)
+    if [[ $current_path == "/" ]]; then
+      current_path=$(print_icon 'ICON_SLASH')
```

Comment:
The icon is missing in the icons array.
And may I suggest a new name for that. I see that `ROOT_ICON` is already in use (and has a different meaning anyway), but what about `ROOT_FOLDER_ICON`?